src/form-host/form-filler.js

```diff
@@ -155,7 +155,8 @@ const fillPage = async (self, pageAnswer) => {
 };
 
 const fillQuestion = (question, answer) => {
-  if (!answer) {
+  // eslint-disable-next-line eqeqeq
+  if(answer == null) {
```

Comment:
i'm not a huge fan of disabling the eslint rule for the single line.  if the rule is wrong, let's change it.  if the rule is right, let's be follow it and just be explicit (because loose equality isn't particulalry intuitive/readable as you point out).

So if you want to change the rule - what about changing the eslint rule for `eqeqeq` to enable this [allow-null](https://eslint.org/docs/rules/eqeqeq#allow-null)?

If you agree that [loose equity isn't intuitive](https://github.com/microsoft/vscode/issues/99947#issuecomment-643386431) then i think `if (answer === null || answer === undefined)` is verbose, explicity, compliant

Reply:
Lets go with `if (answer === null || answer === undefined)`.  For things that are not particularly intuitive, I think it can be helpful to stay verbose and optimize for future readability.  

The only reason I originally used the loose equity here was because I assumed that maybe it made sense to more experienced JS devs. (Of course the fact that it was banned by the linter should have been an indication that it was not a great idea....)  :facepalm: